lib/kodi.py

```diff
@@ -372,7 +372,7 @@ def fillVideoInfos(embyServer, itemId, itemObj, mediaType, item, libraryView='',
             'rating': itemObj.get(PROPERTY_ITEM_COMMUNITY_RATING, 0.0),
             'mpaa': Api._mapMpaa(itemObj.get(PROPERTY_ITEM_OFFICIAL_RATING, '')),
             'duration': Api.ticksToSeconds(itemObj.get(PROPERTY_ITEM_RUN_TIME_TICKS, 0)),
-            'playcount': userdata.get(PROPERTY_ITEM_USER_DATA_PLAY_COUNT, 0),
+            'playcount': userdata.get(PROPERTY_ITEM_USER_DATA_PLAY_COUNT, 0) if userdata.get(PROPERTY_ITEM_USER_DATA_PLAYED) else 0,
```

Comment:
Is `PROPERTY_ITEM_USER_DATA_PLAYED` guaranteed to be there always? Otherwise I would use `if userdata.get(PROPERTY_ITEM_USER_DATA_PLAYED, False) else 0`.

Reply:
As far as I can see it's always there. If it's not it's going to return `None` and this will result in `0`, too.

Reply:
You're right.

Reply:
I just went through the whole code and tried to follow `pylint`, `flake8` and `bandit` code rules. Can you make sure this line is not longer than 120 characters?

I'm going to try to use Github actions to automatically check PRs for code rule violations.

Reply:
Done.